packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -32,26 +33,49 @@ const $occlusionTexture = Symbol('occlusionTexture');
 const $emissiveTexture = Symbol('emissiveTexture');
 const $backingThreeMaterial = Symbol('backingThreeMaterial');
 const $applyAlphaCutoff = Symbol('applyAlphaCutoff');
+export const $lazyLoadGLTFInfo = Symbol('lazyLoadGLTFInfo');
+const $initialize = Symbol('initialize');
+export const $ensureLoaded = Symbol('ensureLoaded');
+
+export enum MaterialLoadMethod {
```

Comment:
Where is this used?

Reply:
Oh, it was from a different time, a simpler time.
Removed